.ci/jsoref-spellchecker/whitelist.words

```diff
@@ -883,7 +883,9 @@ nonnull
 nonrequiredjavadoc
 nonvalidating
 noparentfile
+NOPMD
```

Comment:
hello @AkMo3 , thank you for your PR, 
could you please clarify, why did you put words here? they seem redundant here

Reply:
To show examples from http://roman-ivanov.blogspot.com/2013/10/trailing-comment-check-false-positives.html as mention in the issue